### DISCUSSION TITLE
Editieren - Massenattributierung

### DISCUSSION THREAD

**Discussion by marko-jer** [e-netze/webgis-community#92] [General]
Die Massenattributierung beim webgis-Editeren scheint nicht mehr zu funktionieren - folgende Fehlermeldung wird angezeigt:

![image](https://github.com/gis-eni/webgis-forum/assets/1842500/62d0bdd2-5fda-46aa-8df3-5f935822f7e0)

wenn man beim Bearbeiten von Objekten über "mit Auswahlwerkzeug auswählen" --> "Bearbeiten (edit)" --> "Massenattributierung" klickt.

Zur Info und soweit ich testen konnte:

- mit Build 5.23.0901 hat es noch funktioniert
- mit Build 5.23.1801 kommt dann schon die Fehlermeldung

**Comment by gis-eni** [comment]
Konnte den Bug nachvollziehen. Das kann auftreten, wenn für eine EditMaske Attribute in unterschiedlichen (aufklappbaren) Kategorien anbietet.

Wird mit den nächsten Build 5.23.28xx behoben.
